mbed_flasher/flashers/FlasherMbed.py

```diff
@@ -160,7 +160,27 @@ def flash(self, source, target, method, no_reset):
                 return EXIT_CODE_FILE_COULD_NOT_BE_READ
 
             self.logger.debug("copy finished")
-            sleep(4)
+
+            def worker(mp):
+                mount_point_previous_mounted = None
+
+                count = 10
+
+                while count > 0:
+                    print os.path.ismount(mp)
+
+                    if os.path.ismount(mp):
+                        mount_point_previous_mounted = True
+
+                    if not os.path.ismount(mp) and mount_point_previous_mounted is True:
+                        break
+
+                    sleep(1)
+                    count -= 1
+
+            t = Thread(target=worker, args=(mount_point,))
```

Comment:
I don't think this requires own thread. Nothing is done simultaneously.

Reply:
True. No threading needed here in my opinion.

Reply:
you're right, no need to dothat, would make it more clear,